src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -905,17 +908,25 @@ export default {
 			if (this.isMatchingCampaign) {
 				return this.pageSettingData?.matchingAccountName ?? null;
 			}
-			return this.promoData?.promoFund?.displayName ?? null;
+			const promoCampaignNameFallback = this.promoData?.promoFund?.displayName ?? null;
+			return this.prioritizedTargetCampaignCredit?.promoFund?.displayName ?? promoCampaignNameFallback;
 		},
 		verificationRequired() {
-			if (this.promoData?.managedAccount?.isEmployee && this.promoData?.managedAccount?.formId) {
+			if (this.promoData?.managedAccount?.isEmployee
+				&& this.promoData?.managedAccount?.formId
+				// If promo guest checkout is enabled, we don't need formassembly verification
+				// because the promo guest checkout workflow will take care of email verification.
+				&& !this.promoGuestCheckoutEnabled) {
```

Comment:
@michelleinez I don't think this is entirely true and we may need to have an extra check here. The original check for formId was b/c that was a requirement in order to execute verification. If we're still going to allow form assembly for some accounts we'll probably need an additional computed property to signify which type of verification the account will use. ie. maybe something like `useFormAssebmly()` which checks for the formId. 

Additionally, the `isEmployee` fields tells us if verification is required. In otherwords we shouldn't be showing an email collection field for existing users if `isEmployee` is false. For Guest users we would need to skip the email verification check if `isEmployee` is false.

Reply:
I think we should begin to default to using the new email verification when `isEmployee` is true, activating our new inputs and api check when there is no formId returned on the managed account.

Reply:
Why can't we just disable the contentful flag to make the UI revert to the old behavior, in which case this check would fail because promoGuestCheckout would be disabled?

Reply:
That's what would re-enable formassembly.  So if we need formassembly, we just do that...

Reply:
We'll also do some more work here to more fully deprecate FA in the near future, but for now the feature flag gates the alternative.